backend/src/controllers/usersController/index.ts

```diff
@@ -26,6 +26,37 @@ export default class UsersController {
     res.json({ user })
   }
 
+  @Get('/team/export')
+  public async exportTeam(
+    @RequestDecorator() req: IUserEnhancedRequest,
+    @ResponseDecorator() res: Response
+  ) {
+    const teamId = req.user.team_id
+    const users = await this.userService.getTeamInfo(teamId)
+    const fields = [
```

Comment:
This can be moved to some consts as we don't need to redefine this array with each request